consensus/raft/logging.go

```diff
@@ -0,0 +1,35 @@
+package raft
```

Comment:
A general question: does this refactoring somehow relate to the new config structure, or is it just general cleanup?

Reply:
It's general cleanup. These are just moved because they were in the top of the consensus.go file blocking my view of other things.